pkg/controller/s3/bucket/taggingConfig.go

```diff
@@ -81,13 +80,7 @@ func GenerateTagging(config *v1beta1.Tagging) *awss3.Tagging {
 	if config == nil || config.TagSet == nil {
 		return &awss3.Tagging{TagSet: make([]awss3.Tag, 0)}
```

Comment:
Not changed in this PR, but `nil` is preferred if we know for sure that the array is empty, unless SDK has some weird behavior change between 0-length array vs `nil`.